modules/main.py

```diff
@@ -72,6 +72,7 @@
 Parameters:
     - n     Number of agents
     - e     Number of episodes
+    - E     Max online episodes (in case of no manual user interrupt)
```

Comment:
We reconsidered this, as `e` and `E` have the same meaning but different significance at different tests. I was removed in my opinion.